webapp/src/components/ExpandedSidebar.js

```diff
@@ -154,6 +154,60 @@ function AuctionTableRow({ auction, className, onLandClick }) {
   )
 }
 
+function Districts({ districts }) {
+  if (!districts || districts.data.length === 0) return null
```

Comment:
⚠️ 

data could be undefined here! for example when the districts are loading and/or have an error.
Maybe this can be added as another clause below?

I managed to blow things up going to the stats page and then using the back button

Reply:
Nice catch!